### DISCUSSION TITLE
what about  noaa 9  and metop-b  and sstv  from iss , seeds II , sproud, hades etc .....

### DISCUSSION THREAD

**Discussion by r00krz** [jekhokie/raspberry-noaa-v2#871] [Ideas]
what about to add some satellites for example  noaa 9  and metop-b  and sstv  from iss , seeds II , sprout, hades etc ???

**Comment by RideMyByte** [comment]
"MetOp and FengYun-3 are both polar orbiting satellites that beam back high resolution weather satellite images. Unlike the NOAA polar orbiting satellites which transmit both the easy to receive APT and more advanced HRPT signal, these only transmit a HRPT signal at ~1.70 GHz, so a satellite dish and motorized tracking mount (or hand tracked) is required. You will also need an SDR capable of receiving over 3 MHz bandwidth"
https://www.rtl-sdr.com/standalone-windows-fengyun-3-metop-ahrpt-weather-satellite-decoder/
